### DISCUSSION TITLE
Install advice for Synology NAS

### DISCUSSION THREAD

**Discussion by MuadDibIII** [firefly-iii/firefly-iii#6520] [Firefly III]
Hello, 

When searching for a budget software that I can use at home (PC & mobile), I found Firefly and I was whiling to give it a try.

To do it, I was searching to install it on my Synology Docker, but every tutorial that I found online uses the jc5x/firefly-iii image, and if I understood well it is not maintained anymore, and maybe it was replaced my 3 other containers (core, base, importer).

Is it possible to point me to the right direction in order to install it on my Synology?

Thanks in advance for your return,

Best regards,
Marc

**Comment by JC5** [comment]
The only thing that has changed is that you need this image: `fireflyiii/core:latest`. So if you replace that, you should be able to follow the installation instructions just fine.

**Reply by MuadDibIII** [reply]
Ok, thank you :)
